class.jetpack-gutenberg.php

```diff
@@ -89,6 +94,47 @@ class Jetpack_Gutenberg {
 	 */
 	private static $availability = array();
 
+	/**
+	 * Check to see if a block is able to run with current wp or gutenburg plugin versions
+	 *
+	 * @param array  $version_requirements An array containing wp and plugin version requirements.
+	 * @param string $extension_name The name of the block/plugin that has the version requirement.
```

Comment:
Should we use "slug" instead of "name" here, to keep things clear?

Reply:
Done.